packages/core/tests/unit/services/compression-test.ts

```diff
@@ -59,8 +59,8 @@ module('Unit | Service | compression', function (hooks) {
     const compressedString = await Service.compressModel(report);
 
     assert.equal(
-      compressedString,
-      encodeURIComponent(compressedString),
+      compressedString.replace(/[+/=]/g, ''),
+      encodeURIComponent(compressedString.replace(/[+/=]/g, '')),
```

Comment:
Maybe we should update the service to return `encodeURIComponent(str)`?

Reply:
I'm not sure what that would break tbh, afraid of double encoding.
this was just a bad test that assumed all base64 characters wouldn't get uri encoded. probably should just be removed.

Reply:
So looks like lz-string actually does do something like this already, it just doesn't use `encodeURIComponent` and has it's own safe list of uri characters that includes `+ and -`

So switched to `encodeURI` in tests instead to better match what the lz-string lib actually does.